scripts/xtensa-build-all.sh

```diff
@@ -8,87 +8,64 @@ BUILD_ROM=no
 BUILD_DEBUG=no
 BUILD_FORCE_UP=no
```

Comment:
@crazoes change looks good to me. Anyhow, for easier review can you please split this patch so that each commit only does one thing!

I think each of your list item in commit description should be a separate commit. Commits should be as separate as possible, and they should work independently of following commits in order to not break git bisection.

Reply:
@dbaluta i have split the commit

Reply:
@marc-hb I feel that we need to add a documentation chapter to SOF with how to contribute. We've seen people with different of styles and it would really. 

I'll add this on my TODO list for the next month. If anyone has some free time please jump on this.